### PR TITLE
[wip] Include submitter email in CSV export

### LINKED ISSUE #1215: If a TI applied on behalf of a client, program admin sees the TI's contact info (perhaps in the CSV export)
If I'm a TI and I apply on behalf of an applicant, I would like the program admin to contact both me *and* the applicant with feedback or further questions. This is because the applicant alone might not be able to answer their questions.

So we should probably include a "Prepared By" field(s)  in the CSV export that shows the TI's name and email address. This way, the program admin knows they can talk to the TI in addition to the applicant.

### PR DESCRIPTION
**Pull request by bion**
### Description
1. stores the email of the application submitter on the application if the submitter is a trusted intermediary
2. includes that email or "Applicant" in the third column of the export CSV

### Checklist
- [x] Created tests which fail without the change (if possible)

### Issue(s)
Fixes #1215 1215

### CODE REVIEW

File: universal-application-tool-0.0.1/conf/evolutions/default/26.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# --- Add a column to applications that records the email address of the submitter
+# --- Also ensure that no global admins can be program admins.
```
Comment (kendrickt) [inline universal-application-tool-0.0.1/conf/evolutions/default/26.sql:2]: (blocking, maybe?) Does this "ensure that no global admins can be program admins" happen here?

File: universal-application-tool-0.0.1/test/services/applicant/ApplicantServiceImplTest.java [hunk 2]
```diff
@@ -40,6 +44,7 @@
   private QuestionDefinition questionDefinition;
   private ProgramDefinition programDefinition;
   private UserRepository userRepository;
+  private UatProfile profile;
```
Comment (kendrickt) [inline universal-application-tool-0.0.1/test/services/applicant/ApplicantServiceImplTest.java:47]: (nit) should this be named more specifically so it is clear that it's a profile with a TI account?

**Review by kendrickt — APPROVED** [review]

File: universal-application-tool-0.0.1/conf/evolutions/default/26.sql (hunk 1 above)
Comment (bion) [reply universal-application-tool-0.0.1/conf/evolutions/default/26.sql:2]: Done.

File: universal-application-tool-0.0.1/test/services/applicant/ApplicantServiceImplTest.java (hunk 2 above)
Comment (bion) [reply universal-application-tool-0.0.1/test/services/applicant/ApplicantServiceImplTest.java]: Done.

**Pull request merged**